### ISSUE TITLE
Using ArcFace as a feature extractor

### ISSUE THREAD

**Issue by mkhoshle** [deepinsight/insightface#1192]
Hi, 

I have downloaded the `LResNet100E-IR, ArcFace@ms1m-refine-v2` model and I am planning to use it as a feature extractor. 
When I list the last 10 layers in the pre-trained network I see the following:
```
['bn1_output',
 'dropout0_output',
 'pre_fc1_weight',
 'pre_fc1_bias',
 'pre_fc1_output',
 'fc1_gamma',
 'fc1_beta',
 'fc1_moving_mean',
 'fc1_moving_var',
 'fc1_output']
```
I need to remove the fully connected layers and I am not sure what do `pre_fc1_weight`,  `pre_fc1_bias`, and
 `pre_fc1_output` do? Is there a sketch of the network available anywhere? Are these layers fully connected layers?
I would really appreciate if you could help me with my questions.

Thanks

**Comment by nttstar** [comment]
Use 'fc1_output'.
The weights of classification layer were already removed.

**Comment by mkhoshle** [comment]
@nttstar when I look at the summary of the pre-trained model I see the following and as it shows the pre_fc1 is a FullyConnected layer and fc1 is a BatchNorm layer. I just want to make sure that this is correct. Basically, if I want to use this model I have to consider `dropout0_output` as the last layer before fully connected layer, right? 

<img width="790" alt="Screen Shot 2020-07-30 at 9 49 29 AM" src="https://user-images.githubusercontent.com/21323137/88938052-82acd880-d24a-11ea-93dd-548dfd4dda71.png">

**Comment by hieule88** [comment]
I dont know how to load it into my model, plz tell me
